pkg/services/forwarder/listener_windows.go

```diff
@@ -0,0 +1,16 @@
+package forwarder
+
+import (
+	"github.com/containers/gvisor-tap-vsock/pkg/sshclient"
+	"net"
+	"net/url"
+)
+
+func listenerNpipe(npipePath string) (net.Listener, error) {
+	npipeURI, err := url.Parse(npipePath)
+	if err != nil {
+		return nil, err
+	}
+
+	return sshclient.ListenNpipe(npipeURI) // override to named pipe
```

Comment:
`ListenNpipe` already returns an error on !win platforms, we don't need to redo the same here.